orc8r/cloud/go/obsidian/swagger/mocks/Specs.go

```diff
@@ -0,0 +1,73 @@
+// Code generated by mockery v1.0.0. DO NOT EDIT.
```

Comment:
Use a gen.go file to automatically generate this, e.g. https://github.com/magma/magma/blob/14449aa94ef81b7d259f63c33de4aa41bb1f3481/orc8r/cloud/go/blobstore/gen.go#L14